docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-create-custom-post-plugin.md

```diff
@@ -0,0 +1,95 @@
+---
+title: How to create a custom post plugin for Data Exchange API
+description: This document describes how to create custom post plugins for the Data Exchange API.
+last_updated: Aug 5, 2024
+template: howto-guide-template
+---
+
+## Prerequisites  
+
+* [Install the Data Exchange API](/docs/pbc/all/data-exchange/{{page.version}}/install-and-upgrade/install-the-data-exchange-api.html)
+* [Configure the Data Exchange API](/docs/pbc/all/data-exchange/{{page.version}}/configure-data-exchange-api.html)
+* [Sending requests with Data Exchange API](/docs/pbc/all/data-exchange/{{page.version}}/sending-requests-with-data-exchange-api.md)
+
+
+## Create a post plugin
+
+In some cases, you need to perform some actions after importing or changing data through the Data Exchange API.
+
+For some specific requirements, you may need to create custom post plugins. Post plugins allow you to perform some manipulations after data import or data update. 
+To implement custom post plugins, you need to create a new class that implements the `Spryker\Zed\DynamicEntityExtension\Dependency\Plugin\DynamicEntityPostCreatePluginInterface` or `Spryker\Zed\DynamicEntityExtension\Dependency\Plugin\DynamicEntityPostUpdatePluginInterface` interface.
+
+You can see more details in already existing plugins, please check documentation [Install the Data Exchange API + Category Management feature](/docs/pbc/all/data-exchange/{{page.version}}/install-and-upgrade/install-the-data-exchange-api-category-management-feature.html) or [Install the Data Exchange API + Inventory Management feature](/docs/pbc/all/data-exchange/{{page.version}}/install-and-upgrade/install-the-data-exchange-api-inventory-management-feature.html).
+
+
+### Implement the `DynamicEntityPostCreatePluginInterface` interface
+
+For example, we will create a plugin that will call a facade method after data import for a specific entity, it's an abstract example, and you can implement your logic. If you need to implement a plugin for data updates, you need to implement the  `DynamicEntityPostUpdatePluginInterface` interface. 
```

Comment:
[Note] If we'll remove facade method call from the plugin, this text should be adjusted as well.